raiden/ui/cli.py

```diff
@@ -803,35 +808,32 @@ def prompt_account(address_hex, keystore_path, password_file):
         )
         sys.exit(1)
 
-    if not accmgr.address_in_keystore(address_hex):
-        # check if an address has been passed
-        if address_hex is not None:
-            print("Account '{}' could not be found on the system. Aborting ...".format(
-                address_hex))
-            sys.exit(1)
-
-        addresses = list(accmgr.accounts.keys())
+    if not address_hex:
         formatted_addresses = [
-            '[{:3d}] - {}'.format(idx, to_checksum_address(addr))
-            for idx, addr in enumerate(addresses)
+            to_checksum_address(addr)
+            for addr in accmgr.accounts.keys()
         ]
 
-        should_prompt = True
+        addresses = inquirer.List(
```

Comment:
Can you add an echo above this? To serve the purpose of instructions of how the user selects the address now.

`print('Use the arrow keys to select the address to be used by Raiden')`